src/main/java/org/launchcode/sprinklespre/models/Course.java

```diff
@@ -18,8 +18,22 @@ public class Course extends AbstractEntity{
     @ManyToMany
     private List<User> users = new ArrayList<>();
 
+    //Description of the course
+    private String description;
+
+    //How difficult is the course (1-beginner, 2-intermediate, 3-advanced)
```

Comment:
What do you think about making difficulty an Integer rather than a String? I know generally Integer's are less computationally heavy to work with but that might just be in terms of manipulation.

Reply:
Good call.